README.md

```diff
@@ -448,3 +448,17 @@ template.hbs:
 
 {{load-more-button action='infinityLoad' infinityModel=model}}
 ```
+
+### Delay start of infinite loading until user has indicated they would like to load more
+
+Infinite loading need not start automatically, it may be kickstarted from a user action. The infinity-loader component can be popped in and out, eg: each time a new search is entered, the component may be removed, and then put back in place at the user’s request.
+
+Example:
+
+```hbs
+{{#if hasClickedLoadMore}}
+  {{infinity-loader infinityModel=model triggerOffset=400}}
+{{else}}
+  <button {{action (toggle 'hasClickedLoadMore' this)}}>Load more</button>
```

Comment:
Maybe make this using the `load-more` button like in the example above?  A user would most likely want to load-more when the button is clicked perhaps.  Kind of like how instagram does it...Thoughts?

Reply:
So maybe in your example...
```
{{load-more-button 
  action=(action (pipe (toggle 'hasClickedMore' this) (action 'infinityLoad' model))
}}
```

Yeah using the pipe operator from https://github.com/DockYard/ember-composable-helpers but just detailing how one would do it.

Reply:
And then thinking if you would like, it would be great to also update the example above if you don't mind.  It would be a great change.

```
export default Ember.Component.extend({
  loadText: 'Load more',
  loadedText: 'Loaded',
  click() {    
    this.get('action')();
  }
});
```
&&
```
<ul class="test-list">
{{#each model as |item|}}
  <li>{{item.name}}</li>
{{/each}}
</ul>

{{load-more-button action=(action 'infinityLoad' model)}}
```

Reply:
notice using closure actions instead of sendAction.

Reply:
@gordonbisnor wait I am wrong.  It would totally work the way you had it. 😄  Please ignore all other comments!